src/components/entity/conflict-table.vue

```diff
@@ -247,7 +247,9 @@ defineExpose({ resize });
 
   .unchanged {
     color: #888;
+
     font-style: italic;
+    &:lang(ja), &:lang(zh) { font-style: normal; }
```

Comment:
This is one place where I felt unsure about using bold. In the entity conflict summary table, we italicize values that haven't changed. I'm not sure it makes sense to make them bold in Japanese and Chinese: I'm not sure it conveys the same thing. (If anything, maybe we should bold the values that *have* changed.) Instead, here I'm just removing the italics. The values will still look a little different from values that have changed, because they're also gray.